src/setup_runs/config_read_functions.py

```diff
@@ -227,7 +107,7 @@ def add_environment_variables(config: dict[str, str | bool | int],
     envVarsToInclude = config["environment_variables_for_substitutions"].split(',')
 
     for envVarToInclude in envVarsToInclude :
-        config[envVarToInclude] = environment_variables[envVarToInclude]
+        config[envVarToInclude] = environmental_variables[envVarToInclude]
```

Comment:
I think I fixed this in another PR. Can you find and replace environmental_variables -> environment_variables. environmental_variables isn't grammatically correct here

```suggestion
        config[envVarToInclude] = environment_variables[envVarToInclude]
```

Reply:
I also thought it was fixed. I got all of them with the global find and replace now